lib/trento/infrastructure/commanded/event_handlers/software_updates_discovery_event_handler.ex

```diff
@@ -27,11 +28,21 @@ defmodule Trento.Infrastructure.Commanded.EventHandlers.SoftwareUpdatesDiscovery
     :ok
   end
 
-  def handle(
-        %SoftwareUpdatesDiscoveryCleared{
-          host_id: host_id
-        },
-        _
-      ),
-      do: Discovery.clear_tracked_discovery_result(host_id)
+  def handle(%SoftwareUpdatesHealthChanged{host_id: host_id}, _) do
+    TrentoWeb.Endpoint.broadcast(
```

Comment:
What about wrapping this broadcast in a function? And use that in both `handle`. Yes, just a nit :smile:

Reply:
I disagree in the case of this abstraction. In my view `Endpoint.broadcast/3` is already a nice abstraction provided to us to use - `broadcast(topic, event, payload)`. On top of this, we still need to provide `host_id` to our wrapper function anyway, therefore I'm not sure what work it's really helping us avoid.

Reply:
Up to you, I am fine eitherway :wink: